backend/bitboxbase/bitboxbase.go

```diff
@@ -70,11 +106,26 @@ func (base *BitBoxBase) BlockInfo() string {
 	return base.GetUpdaterInstance().BlockInfo()
 }
 
+//GetIP implements a getter for the bitboxBase ip
+func (base *BitBoxBase) GetIP() string {
+	return base.ip
+}
+
 //Identifier implements a getter for the bitboxBase ID
 func (base *BitBoxBase) Identifier() string {
 	return base.bitboxBaseID
 }
 
+// GetElectrsRPCPort implements a getter for the electrsport
+func (base *BitBoxBase) GetElectrsRPCPort() string {
+	return base.electrsRPCPort
+}
+
+//GetNetwork implements a getter for the network string (either mainnet or testnet)
```

Comment:
I suggest to do `GetCoinCode()` instead and return `"btc"`, `"tbtc"`, as this is what the rest of the app calls it.

You can keep the env api in the base as is, but translate it here to the local glossary.